### PR TITLE
text transition and mode

### LINKED ISSUE #1: Add text transitions
Add text transitions to the existing contents

### LINKED ISSUE #2: Dark mode/light mode
Add dark mode and light mode option in the website

### PR DESCRIPTION
**Pull request by lakshmipriya117**
added text transitions to the content and enabled dark and light mode to the website

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,477 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:400,700');
+body{
```
Comment (LowBP) [inline css/style.css:20]: what is the use of `style2.css` file. if not needed please remove it

File: dark-mode.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```
Comment (LowBP) [inline dark-mode.html]: remove it if not needed

**Comment by LowBP** [comment]
share the website link

File: dark-mode.html (hunk 2 above)
Comment (lakshmipriya117) [reply dark-mode.html]: Have updated the code ,and merged all the css files

File: css/style.css (hunk 1 above)
Comment (lakshmipriya117) [reply css/style.css:20]: Have removed

**Pull request merged**